SimPlatform/SimPlatform.pro

```diff
@@ -24,12 +24,22 @@ CONFIG += c++11
 
 SOURCES += \
         main.cpp \
-        mainwindow.cpp
+        mainwindow.cpp \
+    leftwidget.cpp \
+    rightwidget.cpp
 
-HEADERS += \
-        mainwindow.h
+INCLUDEPATH += "F:/QT/StatLib"
```

Comment:
This path and the one below are only true on your machine. Please use relative paths ../StatLib, or else the code does not build.